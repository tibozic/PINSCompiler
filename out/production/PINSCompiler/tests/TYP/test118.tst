fun f(x: integer, y: logical, z: logical, k: string): integer = ({for y=x,x,x:x},x)
