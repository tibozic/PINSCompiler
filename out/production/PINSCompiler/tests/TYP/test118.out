[1:71-1:72]: ERROR: For loop counter type has to be an integer not `log`

